sdk/compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -1380,71 +1339,6 @@ singleCleanEffect projectOpts =
       whenM (doesPathExist damlArtifactDir) $
         removePathForcibly damlArtifactDir
 
-execPackage :: SdkVersion.Class.SdkVersioned
-            => ProjectOpts
-            -> FilePath -- ^ input file
-            -> Options
-            -> Maybe FilePath
-            -> FromDalf
-            -> Command
-execPackage projectOpts filePath opts mbOutFile dalfInput =
-  Command Package (Just projectOpts) effect
-  where
-    effect = withProjectRoot' projectOpts $ \relativize -> do
-      hPutStrLn stderr $ unlines
-        [ "WARNING: The comannd"
-        , ""
-        , "    daml damlc package"
-        , ""
-        , "is deprecated. Please use"
-        , ""
-        , "    daml build"
-        , ""
-        , "instead."
-        ]
-      loggerH <- getLogger opts "package"
-      filePath <- relativize filePath
-      withDamlIdeState opts loggerH diagnosticsLogger $ \ide -> do
-          -- We leave the sdk version blank and the list of exposed modules empty.
-          -- This command is being removed anytime now and not present
-          -- in the new daml assistant.
-          mbDar <- buildDar ide
-                            PackageConfigFields
-                              { pName = fromMaybe (LF.PackageName $ T.pack $ takeBaseName filePath) $ optMbPackageName opts
-                              , pSrc = filePath
-                              , pExposedModules = Nothing
-                              , pVersion = optMbPackageVersion opts
-                              , pDependencies = []
-                              , pDataDependencies = []
-                              , pSdkVersion = SdkVersion.Class.unresolvedBuiltinSdkVersion
-                              , pModulePrefixes = Map.empty
-                              , pUpgradeDar = Nothing
-                              -- execPackage is deprecated so it doesn't need to support upgrades
-                              }
-                            (toNormalizedFilePath' $ fromMaybe ifaceDir $ optIfaceDir opts)
-                            dalfInput
-                            (optUpgradeInfo opts)
-                            (optDamlWarningFlags opts)
-          case mbDar of
-            Nothing -> do
-                hPutStrLn stderr "ERROR: Creation of DAR file failed."
-                exitFailure
-            Just (dar, _) -> createDarFile loggerH targetFilePath dar
```

Comment:
After this change I think there is only one user of createDarFile left! Good times.